tests/test-async.rs

```diff
@@ -27,12 +27,12 @@ async fn test_async_query() -> Result<()> {
 
     // Act
     let session = client.create_session().await?;
-    let query = r#"CALL SYSTEM$WAIT(120)"#;
+    let query = r#"CALL SYSTEM$WAIT(3)"#;
```

Comment:
Since Snowflake will respond to the query synchronously if the execution time is short enough, could you please revert it to ensure that Snowflake executes it as an asynchronous query?

Reply:
Thanks. Done.